README.md

```diff
@@ -26,7 +26,10 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose person
+  puts "Would you take this rose and help out an old beggar, person?"
+offerRose "young prince"
```

Comment:
We can't call this method inside the method definition. In other words, we should call it *after* the `end` keyword.